### DISCUSSION TITLE
Welcome to Our Discussion！

### DISCUSSION THREAD

**Discussion by Allenkzl** [Seeed-Studio/OSHW-SenseCAP-Watcher#3] [General]
Hello everyone! We're excited to have you here. In this space, we encourage you to share your thoughts, suggestions, and feedback. Whether it’s your insights on the product or ideas for improvement, we’re eager to hear your voice.

Feel free to discuss, ask questions, or share your experiences. We will listen carefully and strive to incorporate your valuable input into our product. Let’s work together to create a better experience!

**Comment by djairjr** [comment]
Obviously, my suggestion has a lot of connection with the hype about Deepseek. I've been using the service for about two months now (during the holiday season and January of this year) and I can say that it works incredibly faster and more accurate than the other AI services I was using before. Is there any initiative to use the app for Watcher using DeepSeek? Is it possible to use Ollama for this? (there are a plethora of tutorials now, explaining how to use DeepSeek's model in Ollama).
